inputremapper/migrations.py

```diff
@@ -155,6 +186,9 @@ def migrate():
     if v < pkg_resources.parse_version("1.3.0"):
         _rename_config()
 
+    if v < pkg_resources.parse_version("1.3.1"):
```

Comment:
This would be in 1.4.0

```suggestion
    if v < pkg_resources.parse_version("1.4.0"):
```